tests/test_self.py

```diff
@@ -34,6 +34,12 @@
 from pylint.reporters.text import BaseReporter, ColorizedTextReporter, TextReporter
 from pylint.utils import utils
 
+if sys.version_info >= (3, 11):
```

Comment:
Should we create a PY311_PLUS constant ?

Reply:
I believe we don't use those on import guards because `mypy` doesn't like them.

Reply:
Right, I forgot